plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -254,7 +260,7 @@
         aliases:
             - eventsListeners
         type: list
-        elements: dict
+        elements: str
```

Comment:
Was it possible to use this option with the old type? If yes, this is a breaking change. If not, it is a regular bugfix.

Reply:
Hi, Felix,
The Keycloak API expects this parameter to be a list of strings (see https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_realmrepresentation). Attempts to call the module with a list of dicts result in HTTP error 500 on calls to create or update API.
As configuring events logging also requires activation through the eventsEnabled parameter which was not allowed by the module, I guess this was a typo and hadn't been tested or reported before.